docs/sql-guide/functions/function-rangeq

```diff
@@ -0,0 +1,58 @@
+---
+title: RANGEQ()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# RANGEQ() function
+
+`RANGEQ()` range function .
+
+## Syntax
+
+```
+rangeq(TQCOLUMN,START,END)
+```
+
+## Arguments
+
+| Argument | Description |
+|---|---|
+| `TQCOLUMN` | Any column that is set with a `time quantum`. |
+
+## Returns
+
+| Data type | Value |
+|---|---|
+| `Timestamp` |  Applies the difference between the 'start' and 'end' timestamps as a filter on the values of the 'time quantum' column. |
+
+
+## Examples
+
+
+
+```sql
+
+CREATE TABLE segments ( 
+    _id id,
+    segment stringsetq timequantum 'YMDH'
+);
+
+insert into segments(_id,segment)
+   VALUES (1, {1789864485,['green', 'yellow']}),
+          (2, {1889763885,['green']}),
+          (3, {1589763885, ['green', 'red']});
+   
+
+select _id, segment from segments where rangeq(segment, 1889763885,2000000000);
```

Comment:
@ljsinclair A user can leave off one of the timestamp arguments with the following syntax: 

select _id, segment from segments where rangeq(segment, 1889763885,null); OR 

select _id, segment from segments where rangeq(segment, null ,2000000000)